public/views/partials/lib/flash/pop.liquid

```diff
@@ -0,0 +1,14 @@
+{% comment %}
+  returns and reset flash messages
+{% endcomment %}
+{% liquid
+  if context.session.sflash != blank
```

Comment:
should we have a namespace for core module? like `context.session.core_modules.sflash` ?

Reply:
can we forget about `sflash` and make it a variable?
then in the app we can call it `sflash` or anything else